hotham/src/contexts/input_context.rs

```diff
@@ -322,6 +322,28 @@ impl RightInputContext {
     }
 }
 
+#[derive(Debug, Default)]
+/// Input from the Head Mounted Display (HMD, or headset)
+pub struct HmdInputContext {
+    left_eye_in_stage: Affine3A,
+    right_eye_in_stage: Affine3A,
```

Comment:
This is a transform, not a point, so it should be `right_eye_from_stage` or `stage_from_right_eye`.

Reply:
I disagree - it is a point. Nothing is ever transformed from that space, and it makes it _very clear_ what this represents.

Reply:
Position does not include orientation. It is very confusing to overload well established terms, see https://en.wikipedia.org/wiki/Position_(geometry). "Pose" is a better term if you want to avoid "transform", it is also used heavily within OpenXR. Yes, I know rapier uses "position" instead of pose for some reason but that does not mean it is a wide spread use. Let's take a more thorough look:

**OpenXR** uses pose to mean orientation and position. https://registry.khronos.org/OpenXR/specs/1.0/html/xrspec.html#XrPosef a pose has a XrVector3f position.

**Unity** separates position from rotation and scale and calls it pose or transform when combined:
https://docs.unity3d.com/ScriptReference/Rigidbody.html - a rigid body has a position and a rotation. You can also get this as a transform, https://docs.unity3d.com/ScriptReference/Transform-position.html - a transform has a Vector3 position. https://docs.unity3d.com/ScriptReference/Pose.html - a pose has a Vector3 position.

**Unreal** separates position and orientation and calls it pose or transform when combined:
https://docs.unrealengine.com/5.0/en-US/BlueprintAPI/SteamVR/GetTrackedDevicePositionandOrien-

**Bevy** uses Transform consistently in the code but sometimes writes position in the docs: https://docs.rs/bevy/latest/bevy/transform/components/struct.Transform.html

**Godot** uses transform consistently.https://docs.godotengine.org/en/stable/tutorials/math/vector_math.html#doc-vector-math and https://docs.godotengine.org/en/stable/tutorials/3d/using_transforms.html#transforms-are-your-friend

**Blender** uses transform with location, rotation, scale. It uses pose for skeletal rigging and animations.

But **rapier** uses position to mean both translation and orientation 🤦 https://rapier.rs/docs/user_guides/rust/rigid_bodies/#position

Reply:
Okay, that is some *next level* research. I really, really appreciate that @rasmusgo. I am now convinced that position is not the best name. 😄 

`left_eye_in_stage` is a good compromise, that plus the type and its usage makes it very easy. We can add a doc string if we're feeling crazy. 

I'm currently on holiday, so if you could please make the suggested change and approve the PR, I'll merge it in. :)